src/Accelerate/vImageTypes.cs

```diff
@@ -271,12 +271,12 @@ public static vImageError ConvolveWithBiasARGBFFFF (ref vImageBuffer src, ref vI
 									Pixel8888 backgroundColor, 
```

Comment:
This file isn't in `API_SOURCES`, so according to your PR description it should get `#nullable enable` added

Reply:
Ah good catch, thank you!